packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -54,7 +54,7 @@ module.exports = class LogPlugin extends Plugin {
   configure (config) {
     return super.configure({
       ...config,
-      enabled: config.enabled && config.logInjection
+      enabled: config.enabled && (config.logInjection || config.ciVisAgentlessLogSubmissionEnabled)
```

Comment:
we also want to log injection if we're automatically submitting logs from test visibility